### ISSUE TITLE
Javascript support in zul editor

### ISSUE THREAD

**Issue by uhegel** [jumperchen/zkidea#22]
It seems that there are no benefits for javascript tags in zul file (syntax highlighting,auto intend, bracket checking…). Is the only way to do this in extra JS file?

**Comment by hawkchen** [comment]
yes, I suggest writing js in a script tab is just for simple and short programs. Regrading a larger js program, suggest to write it in a separate file and include it with `src` attribute.
